### PR TITLE
Creating Features Overview doc

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: overview/features.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+5. **Slash commands**
+
+.. image:: ../images/slash_commands.png
+
+6. **Plug-in framework**
+
+.. image:: ../images/plugin_framework.png
+
+7. **Mobile applications**
```
Comment (lfbrock) [inline overview/features.rst]: May want to consider adjusting screenshot size 

screenshot 2017-12-15 17 45 10

File: overview/features.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+.. image:: ../images/mobile_apps.png
+
+8. **Desktop applications**
+
+.. image:: ../images/desktop_apps.png
+
+9. **Multi-team support**
```
Comment (lfbrock) [inline overview/features.rst]: May want to consider screenshot size here as well
screenshot 2017-12-15 17 45 15

File: overview/features.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+.. image:: ../images/MFA.png
+
+4. **Enterprise support**
+
+.. image:: ../images/enterprise_edition_support.png
+
+5. **SAML-based SSO**
+
+.. image:: ../images/SAML-based_SSO.png
```
Comment (lfbrock) [inline overview/features.rst]: file name here needs to be fixed
screenshot 2017-12-15 17 50 05

File: overview/features.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+6. **Performance Monitoring**
+
+.. image:: ../images/performance_monitoring.png
+
+7. **High availability**
+
+.. image:: ../images/network_diagram.png
+
+8. **Compliance Reporting**
```
Comment (lfbrock) [inline overview/features.rst]: file name here needs to be fixed to match file

screenshot 2017-12-15 17 50 41

**Comment by lfbrock** [comment]
@amyblais can you also double check the name on the file overview feature? 

I think it might be `docs/overview` instead of `docs/source/overview` , it was in the wrong place when I tested on a local copy

**Comment by amyblais** [comment]
I'll need to change the path, so I'll close this one and open a new PR.

**Pull request closed without merging**